### PR TITLE
Add replay flag to data sent to integrations

### PR DESCRIPTION
**Pull request by macpie**

### CODE REVIEW

**Review by dpezely — APPROVED** [review]

File: src/device/router_device_worker.erl [hunk 1]
```diff
@@ -881,8 +881,10 @@ handle_cast(
                         Frame,
                         Region,
                         PacketTime,
-                        HoldTime
+                        HoldTime,
+                        Replay
                     ),
+                    % TODO
```
[the remark this answers is not in the document]
Comment (macpie) [reply src/device/router_device_worker.erl:887]: left over, removing

[the remark this answers is not in the document]
Comment (macpie) [reply src/device/router_device_worker.erl:887]:
```suggestion
```

File: src/device/router_device_worker.erl [hunk 2]
```diff
@@ -1339,7 +1340,7 @@ do_multi_buy(Packet, Device, FrameAck) ->
 ) ->
     {error, any()}
     | {ok, #frame{}, router_device:device(), SendToChannel :: boolean(),
-        {Balance :: non_neg_integer(), Nonce :: non_neg_integer()}}.
+        {Balance :: non_neg_integer(), Nonce :: non_neg_integer()}, boolean()}.
```
Comment (michaeldjeffrey) [inline src/device/router_device_worker.erl:1343]:
```suggestion
        {Balance :: non_neg_integer(), Nonce :: non_neg_integer()}, Replay :: boolean()}.
```

File: src/device/router_device_worker.erl [hunk 3]
```diff
@@ -1392,12 +1393,13 @@ validate_frame(
     Region :: atom(),
     Device :: router_device:device(),
     OfferCache :: map(),
-    Blockchain :: blockchain:blockchain()
+    Blockchain :: blockchain:blockchain(),
+    Replay :: boolean()
 ) ->
     {ok, #frame{}, router_device:device(), SendToChannel :: boolean(),
-        {Balance :: non_neg_integer(), Nonce :: non_neg_integer()}}
+        {Balance :: non_neg_integer(), Nonce :: non_neg_integer()}, boolean()}
```
Comment (michaeldjeffrey) [inline src/device/router_device_worker.erl:1400]:
```suggestion
        {Balance :: non_neg_integer(), Nonce :: non_neg_integer()}, Replay :: boolean()}
```

**Review by michaeldjeffrey — APPROVED** [review]
2 dialyzer updates

**Pull request merged**